docs/conf.py

```diff
@@ -31,33 +31,41 @@
 
 # Add any Sphinx extension module names here, as strings. They can be
 # extensions coming with Sphinx (named 'sphinx.ext.*') or your custom ones.
-extensions = ['sphinx.ext.autodoc', 'sphinx.ext.viewcode']
 
+import sphinx_rtd_theme
+
+extensions = [
+    "sphinxcontrib.apidoc",
+    'sphinx.ext.autodoc',
+    "sphinx_autodoc_typehints",
+    "sphinx.ext.viewcode",
+    "sphinx_rtd_theme",
+    "m2r2",
+]
 # Add any paths that contain templates here, relative to this directory.
 templates_path = ['_templates']
 
 # The suffix(es) of source filenames.
 # You can specify multiple suffix as a list of string:
 #
-# source_suffix = ['.rst', '.md']
-source_suffix = '.rst'
+source_suffix = ['.rst', '.md']
 
 # The master toctree document.
 master_doc = 'index'
 
 # General information about the project.
-project = 'Minos Microservice SAGA'
-copyright = "2021, Clariteia Devs"
+project = 'Minos Microservice Common'
```

Comment:
I think this would be referred to the `minos-microservice-saga` repo, I am right?